maven-plugin/src/main/java/org/openmrs/maven/plugins/SetupSDK.java

```diff
@@ -13,10 +14,17 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.nio.file.Path;
+import java.util.Properties;
+
+import static org.openmrs.maven.plugins.utility.PropertiesUtils.getSdkProperties;
+import static org.openmrs.maven.plugins.utility.PropertiesUtils.savePropertiesChangesToFile;
 
 @Mojo(name = "setup-sdk", requiresProject = false)
 public class SetupSDK extends AbstractTask {
 
+	@Parameter(defaultValue = "false", property = "enableDataSaving")
+	private boolean enableDataSaving;
```

Comment:
I feel like we should change the above name to something that reflects what exactly the effects of this flag will be on the actual implementation.